server/client_message.go

```diff
@@ -8,8 +8,9 @@ import (
 )
 
 type clientMessage struct {
-	Type string          `json:"type"`
-	Data json.RawMessage `json:"data,omitempty"`
+	Type    string          `json:"type"`
+	Data    json.RawMessage `json:"data,omitempty"`
+	mapData map[string]any
```

Comment:
I assume this can be removed?

Reply:
Yep, don't know why that was still there (part of an experiment).